test/functional/wallet_resendwallettransactions.py

```diff
@@ -2,31 +2,69 @@
 # Copyright (c) 2017-2018 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test resendwallettransactions RPC."""
+"""Test that the wallet resends transactions periodically."""
+from collections import defaultdict
+import time
 
+from test_framework.blocktools import create_block, create_coinbase
+from test_framework.messages import ToHex
+from test_framework.mininode import P2PInterface
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import assert_equal, assert_raises_rpc_error
+from test_framework.util import assert_equal, wait_until
+
+class P2PStoreTxInvs(P2PInterface):
+    def __init__(self):
+        super().__init__()
+        self.tx_invs_received = defaultdict(int)
+
+    def on_inv(self, message):
+        # Store how many times invs have been received for each tx.
+        for i in message.inv:
+            if i.type == 1:
+                # save as hex
+                self.tx_invs_received[format(i.hash, 'x')] += 1
 
 class ResendWalletTransactionsTest(BitcoinTestFramework):
```

Comment:
In commit "[tests] Add test for wallet rebroadcasts" (f2faae27895927bb8fefca3d066bf8972e3bdaa7)

It would be nice to move some of the comments from your commit message into the test itself to convey what the test currently covers and how it can be improved in the future ("It does not test whether transactions are actually rebroadcast, or whether the rebroadcast logic is called on a timer").

Reply:
I think you've misunderstood the commit message. I'm saying the test _before_ this commit doesn't test whether txs are actually rebroadcast or whether rebroadcast logic is called on a timer. The commit in this PR fixes that.

Reply:
> I think you've misunderstood the commit message

Oh, yes I did misunderstand. I thought it was saying the test didn't check whether transactions were rebroadcast on an ongoing basis. Anyway, no need to change anything.